server/pages/badge.jsx

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import config from 'config';
+
+const BranchName = ( { branchName, commitChecksum } ) => (
+	<span className={ 'environment branch-name' } title={ 'Commit ' + commitChecksum }>
+		{ branchName }
+	</span>
+);
+
+const DevDocsLink = ( { devDocsUrl, docs } ) => (
+  <span className="environment is-docs">
```

Comment:
Now to be caught by pre-commit hooks and CircleCI 😄 

https://github.com/Automattic/wp-calypso/pull/8565